packages/tasit-action/src/testHelpers/helpers.js

```diff
@@ -64,13 +53,22 @@ const privateKeys = [
   "0xda1a8c477afeb99ae2c2300b22ad612ccf4c184564e332ae9a32f784bbca8d6b",
   "0x633a290bcdabb9075c5a4b3885c69ce64b4b0e6079eb929abb2ac9427c49733b",
 ];
-const accounts = privateKeys.map(createFromPrivateKey);
+export const accounts = privateKeys.map(createFromPrivateKey);
+
+export const waitForEthersEvent = async (eventEmitter, eventName, callback) => {
+  return new Promise((resolve, reject) => {
```

Comment:
It could be interesting to convert this to async/await style too

Reply:
Done on PR https://github.com/tasitlabs/TasitSDK/pull/386

Reply:
Great, thanks! Resolving this